hail/src/main/scala/is/hail/annotations/RegionValueBuilder.scala

```diff
@@ -436,12 +441,14 @@ class RegionValueBuilder(var region: Region) {
           }
 
         case t: TBaseStruct =>
-          a match {
-            case ur: UnsafeRow if currentType() == ur.t =>
-              addUnsafeRow(ur.t, ur)
-            case r: Row =>
-              addRow(t, r)
-          }
+//          a match {
+//            case ur: UnsafeRow if currentType() == ur.t =>
+//              addUnsafeRow(ur.t, ur)
+//            case r: Row =>
+//              addRow(t, r)
+//          }
+          currentType().unstagedStoreJavaObjectAtAddress(currentOffset(), a, region)
```

Comment:
I think this should be the only implementation of addAnnotation!

Reply:
I agree, it will be! I have a `return` on line 417 right now, I just haven't removed the old code yet.

Reply:
oh, great!